os/coreos-layering.md

```diff
@@ -0,0 +1,49 @@
+# CoreOS Layering
+
+This enhancement proposes:
+
+- A fundamental new addition to ostree/rpm-ostree, which is support
+  for directly pulling and updating the OS from container images (while keeping
+  all existing functionality, per-node package layering and
+  our existing support for pulling via ostree on the wire).
+- Documentation for generating derived (layered) images from the pristine CoreOS
+  base image.
+- Support for switching a FCOS system to use a custom image
+  on firstboot via Ignition
+- zincati will continue to perform upgrades by inspecting the upgrade graph
+  from the base image.
```

Comment:
Would be good to expand this bit in a lower section to clarify how it interacts with derivation support.